api/http/api_useradm_test.go

```diff
@@ -772,8 +772,8 @@ func TestUserAdmApiPostVerify(t *testing.T) {
 			nil)
 
 		// set these to make the middleware happy
-		req.Header.Add("X-Original-URI", "/api/mgmt/0.1/someservice/some/resource")
-		req.Header.Add("X-Original-Method", "POST")
+		req.Header.Add("X-Forwarded-Uri", "/api/mgmt/0.1/someservice/some/resource")
```

Comment:
Curious:
Why lowercaps `Uri` ?

Reply:
afaik that is in line with https://en.wikipedia.org/wiki/List_of_HTTP_header_fields#Common_non-standard_response_fields and (which is why I did it): https://github.com/containous/traefik/blob/master/pkg/middlewares/forwardedheaders/forwarded_header.go#L12-L38 -  I guess due to the more common occurrence of things like _HTTPURLConnection_ which might get a bit more readable as _HttpUrlConnection_ - but that's just my guess...